integrationTests/chainSimulator/staking/jail_test.go

```diff
@@ -90,6 +89,9 @@ func testChainSimulatorJailAndUnJail(t *testing.T, targetEpoch int32, nodeStatus
 	})
 	require.Nil(t, err)
 	require.NotNil(t, cs)
+	defer func() {
```

Comment:
I guess we will need to call this on all existing tests. Will start a new PR for this :+1:

Reply:
yes.